tools/cythonize.py

```diff
@@ -40,12 +40,6 @@
 DEFAULT_ROOT = 'numpy'
 VENDOR = 'NumPy'
 
-# WindowsError is not defined on unix systems
-try:
-    WindowsError
-except NameError:
-    WindowsError = None
-
```

Comment:
WindowsError is also an alias of OSError, on Windows only. Also, this exception type is not used anywhere else.